### PR TITLE
Fix The HTML `<Meter>` element example #34413

### LINKED ISSUE #34413: <meter> examples have confusing meter bars inline
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meter

### What specific section or headline is this issue about?

Examples: Simple Example: HTML: first example shown

### What information was incorrect, unhelpful, or incomplete?

> I see:
image
This is the HTML from the example:

```html
<p>
  Heat the oven to <meter min="200" max="500" value="350">350 degrees</meter>.
</p>
```

### What did you expect to see?

I suspect the page author intended the example to render as:
image
I produced this output by moving the closing tag:
```html
<p>
  Heat the oven to <meter min="200" max="500" value="350"></meter> 350 degrees.
</p>
```
This tag closure is more consistent with the 2nd example in the same section on the same page.
I'm not entirely certain what the example's author intended to happen in the example, but I think its reasonable to assume the text "350 degrees." was meant to be displayed after the meter.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

I am using MS Edge (Version 125.0.2535.85 (Official build) (64-bit)) to view the MDN examples. I don't currently have access to other browsers for comparison. Sorry I couldn't be more helpful. This is my first Git issue submission.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/meter`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meter
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/meter/index.md
* Last commit: https://github.com/mdn/content/commit/0f4b28bdc51e89cd25d132b9db12e3e903a9c5aa
* Document last modified: 2024-04-05T16:12:20.000Z

</details>

### PR DESCRIPTION
**Pull request by Ashish-CodeJourney**
### Description

Fix the HTML example for the `<meter>` element to correctly display the text after the meter.

### Motivation

The current example didn't render as intended, potentially confusing readers about the proper usage of the `<meter>` element. This change provides a more accurate and helpful example.

### Additional details

The issue was observed in Google Chrome. The current HTML:

```html
<p>
  Battery level: <meter min="0" max="100" value="75">75%</meter>
</p>
```

should be changed to:

```html
<p>
  Battery level: <meter min="0" max="100" value="75"></meter> 75%
</p>
```

This change is consistent with clearer rendering and better aligns with usage expectations.

### Related issues and pull requests

Fixes #34413

### CODE REVIEW

**Comment by Josh-Cena** [comment]
This is not what https://github.com/mdn/content/issues/34413#issuecomment-2190316590 said should be done, but I'll let @estelle decide.

File: files/en-us/web/html/element/meter/index.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
 
 ```html
 <p>
-  Heat the oven to <meter min="200" max="500" value="350">350 degrees</meter>.
+  Heat the oven to <meter min="200" max="500" value="350"></meter> 350 degrees.
```
Comment (estelle) [inline files/en-us/web/html/element/meter/index.md:45]: The original was intended. The text between the opening and closing tag was put in there for browsers that did not support the `<meter>` element . Now that `<meter>` is supported everywhere, the text can be removed.

The text should not be put after the element as the meter value may change, and the text has no way of being updated to reflect any change.

Comment (estelle) [reply files/en-us/web/html/element/meter/index.md:45]: As @Josh-Cena wrote in the issue, a good solution would be to either replace the example with a useful one or to add within the example description that the text is meant as a fallback text for (a) screen readers (b) browsers that don't support meter bars (c) any other cases when the bar doesn't show.

Comment (Ashish-CodeJourney) [reply files/en-us/web/html/element/meter/index.md]: @estelle @Josh-Cena if you can just review mew example, i guess this makes much more sense

File: files/en-us/web/html/element/meter/index.md [hunk 2]
```diff
@@ -52,17 +52,18 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
 
 On Google Chrome, the resulting meter looks like this:
 
-![A screenshot of the meter element in Google Chrome](screen_shot_2020-10-12_at_10.10.53_pm.png)
+![A screenshot of the meter element in Google Chrome](Screenshot_from_2024-06-27_10-40-55.png)
 
-### High and Low range example
+### High and Low Range Example
```
Comment (estelle) [inline files/en-us/web/html/element/meter/index.md:57]:
```suggestion
### High and low range example
```
Links and headers across the site are sentence case (here and below)

File: files/en-us/web/html/element/meter/index.md [hunk 3]
```diff
@@ -52,17 +52,18 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
 
 On Google Chrome, the resulting meter looks like this:
 
-![A screenshot of the meter element in Google Chrome](screen_shot_2020-10-12_at_10.10.53_pm.png)
```
Comment (estelle) [inline files/en-us/web/html/element/meter/index.md:55]: As meter is well supported in all browsers now, we can remove the screenshots altogether and use live examples instead.

**Review by estelle — COMMENTED** [review]
Titles and links on MDN are sentence case, per the MDN writing style guidelines.

Also, meter is well supported. Instead of updating the images, which was done for when meter did not have full support, let's use live examples instead.

**Comment by Ashish-CodeJourney** [comment]
@estelle  thanks for cooperation
i am occupied with some urgent work load thats why i was nopt  to fix that

**Comment by estelle** [comment]
> @estelle thanks for cooperation i am occupied with some urgent work load thats why i was nopt to fix that

I hope everything is ok.
if you merge https://github.com/Ashish-CodeJourney/content/pull/3, it will update this PR and make this one mergeable

**Comment by Ashish-CodeJourney** [comment]
> > @estelle thanks for cooperation i am occupied with some urgent work load thats why i was nopt to fix that
> 
> I hope everything is ok. if you merge [Ashish-CodeJourney#3](https://github.com/Ashish-CodeJourney/content/pull/3), it will update this PR and make this one mergeable

Done, Thanks :)

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Pull request merged**